test/gtest/ucs/test_sock.cc

```diff
@@ -17,6 +17,22 @@ static std::string socket_err_exp_str;
 
 class test_socket : public ucs::test {
 public:
+    static void fill_sockaddr(sa_family_t af, const char *ip_addr,
```

Comment:
Usually, in a `fill_something` func, the first arg is `something`

Reply:
the convention is passing output params after input params